website/project/views/wiki.py

```diff
@@ -0,0 +1 @@
+x`
```

Comment:
wtf is going on with my local branch when I push? Headed to the office in a
minute -- will re-look upon arrival


On Sun, Feb 15, 2015 at 1:16 PM, Steven Loria <notifications@github.com>
wrote:

> In website/project/views/wiki.py
> <https://github.com/CenterForOpenScience/osf.io/pull/1801#discussion_r24724538>
> :
>
> > @@ -0,0 +1 @@
> > +x`
>
> There's still a stray backtick here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/1801/files#r24724538>
> .
>



-- 
*Harry Rybacki* - Developer, Center for Open Science
Research: http://scinet.osf.io
Blog: http://rybacki.io
[image: http://github.com/harryrybacki]
<http://github.com/HarryRybacki> [image:
https://twitter.com/HarryRybacki] <https://twitter.com/HarryRybacki> [image:
http://www.linkedin.com/in/HarryRybacki/]
<http://www.linkedin.com/in/HarryRybacki/> [image:
https://plus.google.com/+HarryRybacki]
<https://plus.google.com/+HarryRybacki>